EagerExecution.swift

```diff
@@ -21,19 +21,23 @@ import CTensorFlow
 /// eager op object not being freed. If called more than once, then a SEGFAULT may occur due to
 /// trying to execute a TensorFlow eager op that has already been freed.
 @usableFromInline
-internal struct TFE_Op {
+internal struct TFE_Op : TensorFlowGraphOperation {
+  @usableFromInline
+  typealias TensorValueHandle = _AnyTensorHandle
   @usableFromInline internal let status: CTFStatus
   @usableFromInline internal let op: CTFEOp
+  @usableFromInline internal let nOutputs: Int
 
   @usableFromInline
-  internal init(_ name: String) {
+  internal init(_ name: String, _ nOutputs: Int) {
     self.status = TF_NewStatus()
     self.op = TFE_NewOp(_ExecutionContext.global.eagerContext, name, status)
+    self.nOutputs = nOutputs
   }
 
   @inlinable @inline(__always)
```

Comment:
I noticed that we're often using `@inlinable @inline(__always)`. Is this just an artifact of GPE, or there's another reason to annotate definitions this way?

Reply:
I'm pretty sure this was just for GPE and can be removed.  @rxwei would know for sure.

Reply:
Yes, all the `@inlinable` and `@inline(__always)` can be removed.